### PR TITLE
WF-62311-Add information about how to insert image into a table cell

### PR DESCRIPTION
**Pull request by AntonihilSahayaraj**
<p>Kindly review the below merge request and let me know your feedback.</p>
<table>
<tbody>
<tr>
<td>
<p>Task Link</p>
</td>
<td>
<p><a href="https://syncfusion.atlassian.net/browse/WF-62311">https://syncfusion.atlassian.net/browse/WF-62311</a></p>
</td>
</tr>
<tr>
<td>
<p>Description</p>
</td>
<td>
<p>Add UG document to insert image into a table cell</p>
</td>
</tr>
<tr>
<td>
<p>Solution</p>
</td>
<td>
<p>Added code snippet to insert image into a table cell in all platforms.</p>
</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Tables.md [hunk 1]
```diff
@@ -966,6 +966,138 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
 
+## Insert Image to Table Cell
+
+You can insert a image to table cell. The following code example illustrates how to insert a image to table cell.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Creates an instance of WordDocument class.
+WordDocument document = new WordDocument();
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Tables.md:977]: Could you please change the code snippet with "using" statement

File: File-Formats/DocIO/Working-with-Tables.md [hunk 2]
```diff
@@ -966,6 +966,138 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Tables/Align-text-within-a-table).
 
+## Insert Image to Table Cell
+
+You can insert a image to table cell. The following code example illustrates how to insert a image to table cell.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Tables.md:971]: Modify the sentence and also get content review approval 

"You can insert an image into table cell by accessing the paragraph in that cell. The following code example illustrates how to insert an image into table cell."

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Tables.md [hunk 3]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Tables/Align-text-within-a-table).
 
+## Insert Image to Table Cell
+
+You can insert an image into table cell by accessing the paragraph in that cell. The following code example illustrates how to insert an image into table cell.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:971]: You can insert an image into the table cell by accessing the paragraph in that cell. The following code example illustrates how to insert an image into a table cell:

File: File-Formats/DocIO/Working-with-Tables.md [hunk 4]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Creates an instance of WordDocument class.
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:980]: //Adds a new table into the Word document.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 5]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:987]: //Adds the image into the cell.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 6]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(Image.FromFile("Image.png"));
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word document.
+    document.Save("Sample.docx", FormatType.Docx);
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Creates an instance of WordDocument class.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:997]: 'Creates an instance of the WordDocument class.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 7]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+Using document As New WordDocument()
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:998]: Using the document As New WordDocument()

File: File-Formats/DocIO/Working-with-Tables.md [hunk 8]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Dim section As IWSection = document.AddSection()
+    'Adds a new table into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1000]: 'Adds a new table into the Word document.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 9]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Dim table As IWTable = section.AddTable()
+    'Specifies the total number of rows and columns.
+    table.ResetCells(2, 2)
+    table(0, 0).AddParagraph().AppendText("Product Name")
+    table(0, 1).AddParagraph().AppendText("Product Image")
+    table(1, 0).AddParagraph().AppendText("Apple Juice")
+    'Adds image to  the cell.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1007]: 'Adds an image to the cell.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 10]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Dim picture As IWPicture = table(1, 1).AddParagraph().AppendPicture(Image.FromFile("Image.png"))
+    picture.Height = 75
+    picture.Width = 60
+    'Saves the Word document.
+    document.Save("Sample.docx", FormatType.Docx)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+ //Creates an instance of WordDocument class.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1017]: //Creates an instance of the WordDocument class.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 11]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1021]: //Adds a new table into the Word document.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 12]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1028]: //Adds the image into the cell.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 13]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+    Stream imageStream = assembly.GetManifestResourceStream("CreateWordSample.Assets.Image.png");
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(imageStream);
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word file to MemoryStream
+    MemoryStream stream = new MemoryStream();
+    await document.SaveAsync(stream, FormatType.Docx);
+    //Saves the stream as Word file in local machine
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1037]: //Saves the stream as a Word file in a local machine

File: File-Formats/DocIO/Working-with-Tables.md [hunk 14]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Save(stream, "Result.docx");
+    //Refer to the following link to save Word document in UWP platform
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1039]: //Refer to the following link to save the Word document in the UWP platform

File: File-Formats/DocIO/Working-with-Tables.md [hunk 15]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    //https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Creates an instance of WordDocument class.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1045]: //Creates an instance of the WordDocument class.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 16]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1049]: //Adds a new table into the Word document.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 17]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1056]: //Adds the image into the cell.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 18]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    FileStream imageStream = new FileStream(@"Image.png", FileMode.Open, FileAccess.ReadWrite);
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(imageStream);
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word document to MemoryStream.
+    MemoryStream stream = new MemoryStream();
+    document.Save(stream, FormatType.Docx);
+    stream.Position = 0;
+    //Download Word document in the browser.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1065]: //Download the Word document in the browser.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 19]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    return File(stream, "application/msword", "Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Creates an instance of WordDocument class.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1071]: //Creates an instance of the WordDocument class.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 20]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1075]: //Adds a new table into the Word document.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 21]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1082]: //Adds the image into the cell.

File: File-Formats/DocIO/Working-with-Tables.md [hunk 22]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+    Stream imageStream = assembly.GetManifestResourceStream("GettingStarted.Assets.Image.png");
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(imageStream);
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word document to MemoryStream
+    MemoryStream stream = new MemoryStream();
+    document.Save(stream, FormatType.Docx);
+    //Save the stream as a file in the device and invoke it for viewing
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1091]: //Save the stream as a file in the device and invokes it for viewing

File: File-Formats/DocIO/Working-with-Tables.md [hunk 23]
```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
...
+    Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Result.docx", "application/msword", stream);
+    //Download the helper files from the following link to save the stream as file and open the file for viewing in Xamarin platform
```
Comment (OnyimbiNelson) [inline File-Formats/DocIO/Working-with-Tables.md:1093]: //Download the helper files from the following link to save the stream as a file and open the file for viewing on the Xamarin platform:

**Review by AntonihilSahayaraj — COMMENTED** [review]
Feedbacks are addressed

File: File-Formats/DocIO/Working-with-Tables.md (hunk 7 above)
Comment (AntonihilSahayaraj) [reply File-Formats/DocIO/Working-with-Tables.md:998]: It's VB code. So it is not possible to change

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**